docs/examples/operational_memory.md

```
@@ -0,0 +1,234 @@
+4. Assemble the weighted response tensor, **center** over the past index to remove the Markovian background, reshape to :math:`\widetilde{V}(c)`, and take the entropy :math:`S_V(c)` of the normalized singular-value weights. Report :math:`R(c)=\exp(S_V(c))` via `result.rank(c)`.
+
+Use {class}`~mqt.yaqs.memory_characterizer.MemoryCharacterizer` for all memory metrics:
+
+```python
+result = mc.characterize(target, cut=c, k=k, preset="balanced")
+result.entropy(c)  # S_V(c)
+result.singular_values(c)  # spectrum of Ṽ(c)
+result.rank(c)  # R(c) = exp(S_V(c))
+```
+
+### Break weights :math:`w_{\alpha,m}`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)